index.html

```diff
@@ -116,11 +410,118 @@ <h2>Big</h2>
     </div>
   </div>
 
-
-
-
-
-
+  <footer>
+    <div class="f-logo">
+      <a rel="noopener" target="_blank" href="https://www.nytimes.com/"><img class='footer-img' src="pictures/nyt.jpg" alt="..."></a>
+    </div>
+    <div id="f-links" class='flex'>
+      <div>
+        <h3>NEWS</h3>
+        <a href="">Home Page</a>
+        <a href="">World</a>
+        <a href="#">Coronavirus</a>
+        <a href="">U.S.</a>
+        <a href="">Politics</a>
+        <a href="">New York</a>
+        <a href="">Business</a>
+        <a href="">Tech</a>
+        <a href="">Science</a>
+        <a href="">Sports</a>
+        <a href="">Obituaries</a>
+        <a href="">Today's Paper</a>
+        <a href="">Corrections</a>
+      </div>
+      <div>
+        <h3>OPINION</h3>
+        <a href="">Today's Opinion</a>
+        <a href="#">Op-Ed Columnists</a>
+        <a href="">Editorials</a>
+        <a href="">Op-Ed Contributors</a>
+        <a href="">Letters</a>
```

Comment:
## Improve Styling
- [ ] Improve the styling in this footer section as well. The content needs to be as centred as with the original page.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/amiraabouhadid/project1/page-styling/index.html)|
|:--:|:--:|
|![footer](https://user-images.githubusercontent.com/22345961/108414502-5d5ed680-723d-11eb-85ef-e7a44ddd178d.png)|![footer-clone](https://user-images.githubusercontent.com/22345961/108414517-62bc2100-723d-11eb-95fd-e7fb50aa75bd.png)|